runtests.py

```diff
@@ -21,4 +21,6 @@
 django.setup()
 
 if __name__ == "__main__":
-    call_command('test', 'widget_tweaks', verbosity=2)
```

Comment:
Is this test setup tweak really necessary for the functionality fix? 

Could this possibly be split into a separate commit and PR?

Reply:
not really!

ok i move off

Reply:
done: PR #80